src/main/java/racing/domain/TryNumber.java

```diff
@@ -4,28 +4,32 @@
 
     private static final int MIN_TRY_NUMBER = 1;
 
-    private final int tryNumber;
+    private int tryNumber;
 
-    public TryNumber(String tryNumber) {
-        this.tryNumber = toInt(tryNumber);
-        isValidTryNumber(this.tryNumber);
+    public TryNumber(int tryNumber) {
+        checkValidTryNumber(tryNumber);
+        this.tryNumber = tryNumber;
     }
 
-    private int toInt(String tryNumber) {
-        try {
-            return Integer.parseInt(tryNumber);
-        } catch (NumberFormatException nfe) {
-            throw new NumberFormatException("[ERROR] 숫자만 입력 해주세요.");
+    private void checkValidTryNumber(final int tryNumber) {
+        if (tryNumber < MIN_TRY_NUMBER) {
+            throw new IllegalArgumentException("[ERROR] 시도 횟수는 최소 1회 이상이어야 합니다.");
         }
     }
 
-    private static void isValidTryNumber(final int tryNumber) {
-        if (tryNumber < MIN_TRY_NUMBER) {
-            throw new IllegalArgumentException("[ERROR] 시도 횟수는 최소 1회 이상이어야 합니다.");
+    public boolean hasNextStep() {
+        if (this.tryNumber == 0) {
+            return false;
         }
+        return true;
+    }
+
+    public void nextStep() {
+        checkValidTryNumber(this.tryNumber);
+        decrease();
```

Comment:
꼼꼼한 예외처리 👍 👍 👍 
하지만 개인적으로 아래와 같이 변경하면 더 좋을 것 같아요! 🙂
```suggestion
        if(this.tryNumber == 0) {
            throw IllegalArgumentException("남은 시도 횟수가 없습니다.");
        }
        this.tryNumber--;
```
이 메서드에서 발생해야하는 에러와 `checkValidTryNumber`의 에러는 다른 의미를 가지고 있다고 생각해요. 🙂 
그리고 간단한 로직을 굳이 decrease라는 메서드로 쪼갤 필요 없는 것 같아요. 🤔